COMMAND: JKQC 3sa1am2dma-310200_10_150.log 3sa1am2dma-310200_10_163.log 3sa1am2dma-310200_10_181.log 3sa1am2dma-310200_10_53.log 3sa1am2dma-310200_10_55.log 3sa1am2dma-310200_1_132.log 3sa1am2dma-310200_11_53.log 3sa1am2dma-310200_12_212.log 3sa1am2dma-310200_1_26.log 3sa1am2dma-310200_13_172.log 3sa1am2dma-310200_14_143.log 3sa1am2dma-310200_14_96.log 3sa1am2dma-310200_16_131.log 3sa1am2dma-310200_16_144.log 3sa1am2dma-310200_17_1.log 3sa1am2dma-310200_17_207.log 3sa1am2dma-310200_17_212.log 3sa1am2dma-310200_17_215.log 3sa1am2dma-310200_17_53.log 3sa1am2dma-310200_17_66.log 3sa1am2dma-310200_17_98.log 3sa1am2dma-310200_2_190.log 3sa1am2dma-310200_22_100.log 3sa1am2dma-310200_22_134.log 3sa1am2dma-310200_3_161.log 3sa1am2dma-310200_3_162.log 3sa1am2dma-310200_3_181.log 3sa1am2dma-310200_3_194.log 3sa1am2dma-310200_3_51.log 3sa1am2dma-310200_3_89.log 3sa1am2dma-310200_3_94.log 3sa1am2dma-310200_8_169.log 3sa1am2dma-310200_8_94.log 3sa1am2dma-310200_9_123.log 3sa1am2dma-310200_9_129.log 3sa1am2dma-310200_9_150.log 3sa1am2dma-310200_9_156.log -reacted -noexample -out database.pkl
COMMAND: JKQC database.pkl -ct -gout
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_1am_2dma
slurmstepd: error: *** JOB 2640004 ON s95n24 CANCELLED AT 2023-03-08T13:48:21 ***
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_1am_2dma
COMMAND: JKQC 3sa1am2dma-310200_10_150.log 3sa1am2dma-310200_10_163.log 3sa1am2dma-310200_10_181.log 3sa1am2dma-310200_10_53.log 3sa1am2dma-310200_10_55.log 3sa1am2dma-310200_1_132.log 3sa1am2dma-310200_11_53.log 3sa1am2dma-310200_12_212.log 3sa1am2dma-310200_1_26.log 3sa1am2dma-310200_13_172.log 3sa1am2dma-310200_14_143.log 3sa1am2dma-310200_14_96.log 3sa1am2dma-310200_16_131.log 3sa1am2dma-310200_16_144.log 3sa1am2dma-310200_17_1.log 3sa1am2dma-310200_17_207.log 3sa1am2dma-310200_17_212.log 3sa1am2dma-310200_17_215.log 3sa1am2dma-310200_17_53.log 3sa1am2dma-310200_17_66.log 3sa1am2dma-310200_17_98.log 3sa1am2dma-310200_2_190.log 3sa1am2dma-310200_22_100.log 3sa1am2dma-310200_22_134.log 3sa1am2dma-310200_3_161.log 3sa1am2dma-310200_3_162.log 3sa1am2dma-310200_3_181.log 3sa1am2dma-310200_3_194.log 3sa1am2dma-310200_3_51.log 3sa1am2dma-310200_3_89.log 3sa1am2dma-310200_3_94.log 3sa1am2dma-310200_8_169.log 3sa1am2dma-310200_8_94.log 3sa1am2dma-310200_9_123.log 3sa1am2dma-310200_9_129.log 3sa1am2dma-310200_9_150.log 3sa1am2dma-310200_9_156.log -reacted -noexample -out database1DLPNO.pkl
Number of files: 37
COMMAND: JKQC database1DLPNO.pkl -ct -gout
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_1am_2dma
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_1am_2dma
COMMAND: JKQC 3sa1am2dma-310200_10_150.log 3sa1am2dma-310200_10_163.log 3sa1am2dma-310200_10_181.log 3sa1am2dma-310200_10_53.log 3sa1am2dma-310200_10_55.log 3sa1am2dma-310200_1_132.log 3sa1am2dma-310200_11_53.log 3sa1am2dma-310200_12_212.log 3sa1am2dma-310200_1_26.log 3sa1am2dma-310200_13_172.log 3sa1am2dma-310200_14_143.log 3sa1am2dma-310200_14_96.log 3sa1am2dma-310200_16_131.log 3sa1am2dma-310200_16_144.log 3sa1am2dma-310200_17_1.log 3sa1am2dma-310200_17_207.log 3sa1am2dma-310200_17_212.log 3sa1am2dma-310200_17_215.log 3sa1am2dma-310200_17_53.log 3sa1am2dma-310200_17_66.log 3sa1am2dma-310200_17_98.log 3sa1am2dma-310200_2_190.log 3sa1am2dma-310200_22_100.log 3sa1am2dma-310200_22_134.log 3sa1am2dma-310200_3_161.log 3sa1am2dma-310200_3_162.log 3sa1am2dma-310200_3_181.log 3sa1am2dma-310200_3_194.log 3sa1am2dma-310200_3_51.log 3sa1am2dma-310200_3_89.log 3sa1am2dma-310200_3_94.log 3sa1am2dma-310200_8_169.log 3sa1am2dma-310200_8_94.log 3sa1am2dma-310200_9_123.log 3sa1am2dma-310200_9_129.log 3sa1am2dma-310200_9_150.log 3sa1am2dma-310200_9_156.log -reacted -noexample -out database3DFT.pkl
Number of files: 37
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_1am_2dma
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_1am_2dma
COMMAND: JKQC 3sa1am2dma-310200_10_150.log 3sa1am2dma-310200_10_163.log 3sa1am2dma-310200_10_181.log 3sa1am2dma-310200_10_53.log 3sa1am2dma-310200_10_55.log 3sa1am2dma-310200_1_132.log 3sa1am2dma-310200_11_53.log 3sa1am2dma-310200_12_212.log 3sa1am2dma-310200_1_26.log 3sa1am2dma-310200_13_172.log 3sa1am2dma-310200_14_143.log 3sa1am2dma-310200_14_96.log 3sa1am2dma-310200_16_131.log 3sa1am2dma-310200_16_144.log 3sa1am2dma-310200_17_1.log 3sa1am2dma-310200_17_207.log 3sa1am2dma-310200_17_212.log 3sa1am2dma-310200_17_215.log 3sa1am2dma-310200_17_53.log 3sa1am2dma-310200_17_66.log 3sa1am2dma-310200_17_98.log 3sa1am2dma-310200_2_190.log 3sa1am2dma-310200_22_100.log 3sa1am2dma-310200_22_134.log 3sa1am2dma-310200_3_161.log 3sa1am2dma-310200_3_162.log 3sa1am2dma-310200_3_181.log 3sa1am2dma-310200_3_194.log 3sa1am2dma-310200_3_51.log 3sa1am2dma-310200_3_89.log 3sa1am2dma-310200_3_94.log 3sa1am2dma-310200_8_169.log 3sa1am2dma-310200_8_94.log 3sa1am2dma-310200_9_123.log 3sa1am2dma-310200_9_129.log 3sa1am2dma-310200_9_150.log 3sa1am2dma-310200_9_156.log -reacted -noexample -out database.pkl
Number of files: 37
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_1am_2dma
COMMAND: JKQC database.pkl -noexample -fc 100 -v 0.996 -sort g -select 3 -reacted -unique rg,el,g -out database3DFT.pkl
COMMAND: JKQC database.pkl -noexample -fc 100 -v 0.996 -globout -reacted -out database1DLPNO.pkl
COMMAND: JKQC database1DLPNO.pkl -ct -gout
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_1am_2dma
COMMAND: JKQC database.pkl -noexample -fc 100 -v 0.996 -sort gout -select 1 -reacted -out database1DLPNO.pkl
Traceback (most recent call last):
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 3361, in get_loc
    return self._engine.get_loc(casted_key)
  File "pandas/_libs/index.pyx", line 76, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 108, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 5198, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 5206, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'out'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKQCpickle.py", line 2042, in <module>
    sorted_indices = (clusters_df["log"]["gibbs_free_energy"]-clusters_df["log"]["electronic_energy"]+clusters_df["out"]["electronic_energy"]).sort_values().index
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/frame.py", line 3457, in __getitem__
    return self._getitem_multilevel(key)
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/frame.py", line 3508, in _getitem_multilevel
    loc = self.columns.get_loc(key)
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/indexes/multi.py", line 2922, in get_loc
    loc = self._get_level_indexer(key, level=0)
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/indexes/multi.py", line 3204, in _get_level_indexer
    idx = self._get_loc_single_level_index(level_index, key)
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/indexes/multi.py", line 2855, in _get_loc_single_level_index
    return level_index.get_loc(key)
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 3363, in get_loc
    raise KeyError(key) from err
KeyError: 'out'
COMMAND: JKQC database1DLPNO.pkl -ct -gout
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_1am_2dma
COMMAND: JKQC database.pkl -noexample -fc 100 -v 0.996 -sort gout -select 1 -reacted -out database1DLPNO.pkl
COMMAND: JKQC database1DLPNO.pkl -ct -gout
